### PR TITLE
Create the routing and layouts-login-registration

### PR DESCRIPTION
**Pull request by 1810Nikita**
*Linked Work Items*
https://internship-splitwise.atlassian.net/browse/IS-29
*Checklist*
- [x] Code adheres to *[Git and Pull Request Guidelines]*
- [x] Code adheres to *[Angular Guidelines]*
- [x] Code adheres to *[Front-end Checklist]*

### CODE REVIEW

File: src/app/app.component.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.login{
```
Comment (AshaPatel216) [inline src/app/app.component.scss]: Why all this style is here?

If this style is common to all components then move it to styles.scss. And if the style is for individual component then move it to respective SCSS file.

File: src/app/app.component.scss [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    max-width: 200px;
+    margin-left: 20px;
+    text-align: center;
+}
+h1{
+    text-align: center;
+    margin-top: 10px ;
+}
+.wlcome-page{
+    text-align: center;
```
Comment (AshaPatel216) [inline src/app/app.component.scss]: You can use bootstrap class for text center after you start using bootstrap. So please change this in next PR. no need to in this PR

File: src/app/app.component.ts [hunk 3]
```diff
@@ -7,4 +8,15 @@ import { Component } from '@angular/core';
 })
 export class AppComponent {
   title = 'splitwise';
+  showLoginButton = true;
```
Comment (AshaPatel216) [inline src/app/app.component.ts]: Only variable declaration will be here. showLoginBUtton: boolean;

File: src/app/app.component.ts [hunk 4]
```diff
@@ -7,4 +8,15 @@ import { Component } from '@angular/core';
...
+  constructor(private router: Router) {
+    this.router.events.subscribe((val) => {
```
Comment (AshaPatel216) [inline src/app/app.component.ts:16]: The default value initialization of any variable will be inside the constructor.
showLoginButton = false; or showLoginButton = true;

File: src/app/login-signup/login-signup.component.html [hunk 5]
```diff
@@ -1 +1,9 @@
-<p>login-signup works!</p>
+<!--<p>login-signup works!</p>
```
Comment (AshaPatel216) [inline src/app/login-signup/login-signup.component.html]: Remove commented code.

File: src/app/login-signup/login-signup.module.ts [hunk 6]
```diff
@@ -3,19 +3,21 @@ import { CommonModule } from '@angular/common';
 
 import { LoginSignupRoutingModule } from './login-signup-routing.module';
 import { LoginSignupComponent } from './login-signup.component';
-import { LoginComponent } from './login/login.component';
+//import { LoginComponent } from './login/login.component';
```
Comment (AshaPatel216) [inline src/app/login-signup/login-signup.module.ts]: Remove commented code

File: src/app/login-signup/login-signup.module.ts [hunk 7]
```diff
@@ -3,19 +3,21 @@ import { CommonModule } from '@angular/common';
...
 import { RegisterComponent } from './register/register.component';
+import { ReactiveFormsModule } from '@angular/forms';
 
 
 @NgModule({
   declarations: [
     LoginSignupComponent,
-    LoginComponent,
+    //LoginComponent,
```
Comment (AshaPatel216) [inline src/app/login-signup/login-signup.module.ts]: Remove commented code

File: src/app/login-signup/login/login.component.html [hunk 8]
```diff
@@ -1 +1,22 @@
-<p>login works!</p>
+
```
Comment (AshaPatel216) [inline src/app/login-signup/login/login.component.html]: Missing comments in HTML

File: src/app/login-signup/login/login.component.html [hunk 9]
```diff
@@ -1 +1,22 @@
...
+    <div id="Login" class="background">
+
+        <form [formGroup]="loginForm">
+            <h3>Login Here</h3>
+        
+            <label for="username">Email</label><br>
```
Comment (AshaPatel216) [inline src/app/login-signup/login/login.component.html]: Wrap label and input in div tag to make proper sections of each.
Follow this to other sections too

File: src/app/login-signup/login/login.component.html [hunk 10]
```diff
@@ -1 +1,22 @@
...
+            <input type="text" placeholder="Email " id="username" formControlName="username"><br>
+        
+            <label for="password">Password</label><br>
+            <input type="password" placeholder="Password" id="password" formControlName="password"><br>
+        
+            <button class="btn" (click)="onLogin()">Log In</button><br>
```
Comment (AshaPatel216) [inline src/app/login-signup/login/login.component.html]: Missing button type

File: src/app/login-signup/login/login.component.ts [hunk 11]
```diff
@@ -1,10 +1,62 @@
...
 
 @Component({
   selector: 'app-login',
   templateUrl: './login.component.html',
   styleUrls: ['./login.component.scss']
 })
-export class LoginComponent {
+export class LoginComponent implements OnInit{
+  loginForm!: FormGroup;
+  signupUsers:any[] =[];
```
Comment (AshaPatel216) [inline src/app/login-signup/login/login.component.ts]: Initialize the default value in constructor

File: src/app/login-signup/login/login.component.ts [hunk 12]
```diff
@@ -1,10 +1,62 @@
...
+    if(localData != null) {
+      this.signupUsers=JSON.parse(localData)
+    }
+    this.loginForm = new FormGroup({
+      username: new FormControl('', Validators.required),
+      password: new FormControl('', Validators.required)
+    });
+  }
+
+  onLogin() {
```
Comment (AshaPatel216) [inline src/app/login-signup/login/login.component.ts]: Missing return type. Missing comments for method. Please check this to all files and make correction. please follow the angular function comments guidelines.

File: src/app/login-signup/login/login.component.ts [hunk 13]
```diff
@@ -1,10 +1,62 @@
...
+    const username = this.loginForm.get('username')?.value;
+    const password = this.loginForm.get('password')?.value;
+    const isUserExist = this.signupUsers.find(m => m.username == username && m.password == password);
+    if(isUserExist != undefined) {
```
Comment (AshaPatel216) [inline src/app/login-signup/login/login.component.ts]: Use !== instead of !=

File: src/app/login-signup/login/login.component.ts [hunk 14]
```diff
@@ -1,10 +1,62 @@
...
+      alert('user logged in successfully')
```
Comment (AshaPatel216) [inline src/app/login-signup/login/login.component.ts]: Missing semicolon

File: src/app/login-signup/login/login.component.ts [hunk 15]
```diff
@@ -1,10 +1,62 @@
...
+    } else{
+      alert('user not found please Register')
```
Comment (AshaPatel216) [inline src/app/login-signup/login/login.component.ts]: Missing semicolon

File: src/app/login-signup/login/login.component.ts [hunk 16]
```diff
@@ -1,10 +1,62 @@
...
+    }
+  }
+
+  navigateToRegister() {
+    this.router.navigate(['login-signup/register']);
+  }
+    
 
 }
+
```
Comment (AshaPatel216) [inline src/app/login-signup/login/login.component.ts]: Remove extra line of space from line 46 to 62

File: src/app/login-signup/register/register.component.html [hunk 17]
```diff
@@ -1 +1,15 @@
-<p>register works!</p>
+<form [formGroup]="registrationForm">
+    <h3>Register Here</h3><br>
+
+    <label for="username">Email</label><br>
+    <input type="text" placeholder="Email" id="username" formControlName="username"><br>
+
```
Comment (AshaPatel216) [inline src/app/login-signup/register/register.component.html]: Missing Name for registration

File: src/app/login-signup/register/register.component.html [hunk 18]
```diff
@@ -1 +1,15 @@
-<p>register works!</p>
+<form [formGroup]="registrationForm">
```
Comment (AshaPatel216) [inline src/app/login-signup/register/register.component.html]: Check all comments given in login html and make correction

File: src/app/login-signup/register/register.component.ts [hunk 19]
```diff
@@ -1,10 +1,50 @@
...
   selector: 'app-register',
   templateUrl: './register.component.html',
   styleUrls: ['./register.component.scss']
 })
-export class RegisterComponent {
+export class RegisterComponent implements OnInit{
 
+  registrationForm!: FormGroup;
+  signupUsers:any[] =[];
+  signupObj:any ={
```
Comment (AshaPatel216) [inline src/app/login-signup/register/register.component.ts]: Don't use any. Give specific type.

File: src/app/app.component.html [hunk 20]
```diff
@@ -1,484 +1,6 @@
...
-      </svg>
-    </a>
-    <a aria-label="Angular on YouTube" target="_blank" rel="noopener" href="https://youtube.com/angular" title="YouTube">
-      <svg id="youtube-logo" height="24" width="24" data-name="Logo" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="#fff">
-        <path d="M0 0h24v24H0V0z" fill="none"/>
-        <path d="M21.58 7.19c-.23-.86-.91-1.54-1.77-1.77C18.25 5 12 5 12 5s-6.25 0-7.81.42c-.86.23-1.54.91-1.77 1.77C2 8.75 2 12 2 12s0 3.25.42 4.81c.23.86.91 1.54 1.77 1.77C5.75 19 12 19 12 19s6.25 0 7.81-.42c.86-.23 1.54-.91 1.77-1.77C22 15.25 22 12 22 12s0-3.25-.42-4.81zM10 15V9l5.2 3-5.2 3z"/>
-      </svg>
-    </a>
+<!--First page-->
```
Comment (AshaPatel216) [inline src/app/app.component.html]: Add proper comment. what does 'First page convey' ?

File: src/app/app.component.html [hunk 21]
```diff
@@ -1,484 +1,6 @@
...
+<div class="wlcome-page">
```
Comment (AshaPatel216) [inline src/app/app.component.html]: spelling: wlcome -> welcome

File: src/app/app.component.ts [hunk 22]
```diff
@@ -7,4 +8,18 @@ import { Component } from '@angular/core';
 })
 export class AppComponent {
   title = 'splitwise';
+  // Declare showLoginButton property
+  showLoginButton: boolean;
+  constructor(private router: Router) {
+    // Initialize showLoginButton property to true
+    this.showLoginButton=true;
```
Comment (AshaPatel216) [inline src/app/app.component.ts]: Make consistency in adding space before and after '='

File: src/app/login-signup/login/login.component.html [hunk 23]
```diff
@@ -1 +1,22 @@
...
+            <div>
+                <label for="username">Email</label><br>
+                <input type="text" placeholder="Email " id="username" formControlName="username"><br>
+            </div>
+            <!--Add password-->
+            <div>
+                <label for="password">Password</label><br>
+                <input type="password" placeholder="Password" id="password" formControlName="password"><br>
+            </div>
+        </div><br><br>
```
Comment (AshaPatel216) [inline src/app/login-signup/login/login.component.html]: Avoid use of br

File: src/app/login-signup/login/login.component.html [hunk 24]
```diff
@@ -1 +1,22 @@
...
+        <h3>Login Here</h3>
+        <div class="input-fields">
+            <!--Add email-->
+            <div>
+                <label for="username">Email</label><br>
+                <input type="text" placeholder="Email " id="username" formControlName="username"><br>
+            </div>
+            <!--Add password-->
+            <div>
+                <label for="password">Password</label><br>
```
Comment (AshaPatel216) [inline src/app/login-signup/login/login.component.html]: avoid use of br

File: src/app/login-signup/login/login.component.html [hunk 25]
```diff
@@ -1 +1,22 @@
-<p>login works!</p>
+<!--login component-->
+<div id="Login" class="container">
+    <!--login form-->
+    <form [formGroup]="loginForm">
+        <h3>Login Here</h3>
+        <div class="input-fields">
+            <!--Add email-->
+            <div>
+                <label for="username">Email</label><br>
```
Comment (AshaPatel216) [inline src/app/login-signup/login/login.component.html]: same as below

File: src/app/login-signup/login/login.component.html [hunk 26]
```diff
@@ -1 +1,22 @@
...
+            </div>
+            <!--Add password-->
+            <div>
+                <label for="password">Password</label><br>
+                <input type="password" placeholder="Password" id="password" formControlName="password"><br>
+            </div>
+        </div><br><br>
+        <!--Buttons for login and navigation to registration page-->
+        <button type='button' class="btn" (click)="onLogin()">Log In</button><br><br>
+        <button type='button' class="btn" (click)="navigateToRegister()">Register</button>
```
Comment (AshaPatel216) [inline src/app/login-signup/login/login.component.html:20]: same as above

File: src/app/login-signup/login/login.component.ts [hunk 27]
```diff
@@ -1,10 +1,50 @@
-import { Component } from '@angular/core';
 
+import { Component ,OnInit} from '@angular/core';
+import { FormBuilder, FormGroup, Validators , FormControl} from '@angular/forms';
+import { ActivatedRoute,Router } from '@angular/router';
+import { ReactiveFormsModule } from '@angular/forms';
+
+// an interface that contains the properties of username and password.
+interface User {
```
Comment (AshaPatel216) [inline src/app/login-signup/login/login.component.ts:8]: create model for this. do not change right now but while implementing it, please remove interface and create modal

File: src/app/login-signup/login/login.component.ts [hunk 28]
```diff
@@ -1,10 +1,50 @@
...
+}
 @Component({
   selector: 'app-login',
   templateUrl: './login.component.html',
   styleUrls: ['./login.component.scss']
 })
-export class LoginComponent {
+export class LoginComponent implements OnInit{
+  loginForm!: FormGroup;
+  signupUsers: User[] =[];
```
Comment (AshaPatel216) [inline src/app/login-signup/login/login.component.ts]: The value is already initialize in constructor. No need to initialize here.

File: src/app/login-signup/login/login.component.ts [hunk 29]
```diff
@@ -1,10 +1,50 @@
...
+    if(localData != null) {
+      this.signupUsers=JSON.parse(localData);
+    }
+    this.loginForm = new FormGroup({
+      username: new FormControl('', Validators.required),
+      password: new FormControl('', Validators.required)
+    });
+  }
 
+  onLogin(): void {
```
Comment (AshaPatel216) [inline src/app/login-signup/login/login.component.ts:41]: Missing comment. please check how to write multi line comments and make correction

File: src/app/login-signup/login/login.component.ts [hunk 30]
```diff
@@ -1,10 +1,50 @@
...
+    const username:string = this.loginForm.get('username')?.value;
+    const password:string = this.loginForm.get('password')?.value;
+    const isUserExist = this.signupUsers.find(m => m.username === username && m.password === password);
+    if(isUserExist !== undefined) {
+      alert('user logged in successfully');
+    } else{
+      alert('user not found please Register');
+    }
+  }
+  //navigate to register page
```
Comment (AshaPatel216) [inline src/app/login-signup/login/login.component.ts]: Please check how to write comments for functions.

File: src/app/login-signup/login/login.component.ts [hunk 31]
```diff
@@ -1,10 +1,50 @@
...
+  navigateToRegister():void {
```
Comment (AshaPatel216) [inline src/app/login-signup/login/login.component.ts]: Make consistency in spacing
function():void ->  function(): void

File: src/app/login-signup/register/register.component.html [hunk 32]
```diff
@@ -1 +1,30 @@
-<p>register works!</p>
+<div class="container">
+    <!--Registration form-->
+    <form [formGroup]="registrationForm">
+        <h3>Register Here</h3><br>
+        <div class="input-fields">
+            <!--Email-->
+            <div>
+                <label for="username">Email</label><br>
```
Comment (AshaPatel216) [inline src/app/login-signup/register/register.component.html]: Remove all br tag and add proper styling

File: src/app/login-signup/register/register.component.ts [hunk 33]
```diff
@@ -1,10 +1,62 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators , FormControl} from '@angular/forms';
+import { ReactiveFormsModule } from '@angular/forms';
+
+// an interface that contains the properties of username ,name, confirm password and password.
+interface User {
```
Comment (AshaPatel216) [inline src/app/login-signup/register/register.component.ts:6]: Same comment as above .ts file

File: src/app/login-signup/register/register.component.ts [hunk 34]
```diff
@@ -1,10 +1,62 @@
...
+  username: string,
+  name:string,
```
Comment (AshaPatel216) [inline src/app/login-signup/register/register.component.ts]: Make consistency in spacing. check this to all
name:string -> name: string

File: src/app/login-signup/register/register.component.ts [hunk 35]
```diff
@@ -1,10 +1,62 @@
...
 @Component({
   selector: 'app-register',
   templateUrl: './register.component.html',
   styleUrls: ['./register.component.scss']
 })
-export class RegisterComponent {
 
+export class RegisterComponent implements OnInit{
+  registrationForm!: FormGroup;
+  signupUsers:User[] =[];
```
Comment (AshaPatel216) [inline src/app/login-signup/register/register.component.ts]: remove initialization from here

File: src/app/styles/components/button.scss [hunk 36]
```diff
@@ -0,0 +1,20 @@
...
+    border-radius: 5px;
+    cursor: pointer;
+    color: black;
+    background-color: rgb(192, 138, 243) ;
+    border: 2px solid rgb(133, 22, 238);
+    
+}
+
+.btn{
+
```
Comment (AshaPatel216) [inline src/app/styles/components/button.scss]: Remove extra space

File: src/app/styles/components/button.scss [hunk 37]
```diff
@@ -0,0 +1,20 @@
...
+    background-color: rgb(192, 138, 243) ;
+    border: 2px solid rgb(133, 22, 238);
+    
+}
+
+.btn{
+    
+    background-color: rgb(192, 138, 243) ;
+    color: black;
+    border: 2px solid blueviolet;
```
Comment (AshaPatel216) [inline src/app/styles/components/button.scss:18]: Use variable in next PR.

File: src/app/styles/components/button.scss [hunk 38]
```diff
@@ -0,0 +1,20 @@
...
+    color: black;
+    background-color: rgb(192, 138, 243) ;
+    border: 2px solid rgb(133, 22, 238);
+    
+}
+
+.btn{
+    
+    background-color: rgb(192, 138, 243) ;
+    color: black;
```
Comment (AshaPatel216) [inline src/app/styles/components/button.scss:17]: Use variable in next PR.

File: src/styles.scss [hunk 39]
```diff
@@ -2,3 +2,82 @@
 
 /* Importing Bootstrap SCSS file. */
 @import 'bootstrap/scss/bootstrap';
+@import '~@angular/material/prebuilt-themes/indigo-pink.css';
+@import 'app/styles/components/button.scss';
+@import 'app/styles/layout/header.scss';
+
+html, body { height: 100%; }
```
Comment (AshaPatel216) [inline src/styles.scss]: html, body ->
html,
body {

}

File: src/styles.scss [hunk 40]
```diff
@@ -2,3 +2,82 @@
...
+body { margin: 0;
+    background-color: white;
+}
+
+//h1 styling
+h1{
```
Comment (AshaPatel216) [inline src/styles.scss]: Maintain consistency in spacing.
h1{ -> h1 {

follow this to all styles

File: src/styles.scss [hunk 41]
```diff
@@ -2,3 +2,82 @@
...
+    position: absolute;
+    transform: translate(-50%,-50%);
+    top: 50%;
+    left: 50%;
+    border: 2px solid blueviolet;
+    border-radius: 10px;
+    backdrop-filter: blur(10px);
+    box-shadow: 0 0 40px rgba(95, 84, 177, 0.6);
+    padding: 30px 15px;
+    font-family: ‘Poppins’,sans-serif;
```
Comment (AshaPatel216) [inline src/styles.scss:36]: Use variable in next PR

File: src/styles.scss [hunk 42]
```diff
@@ -2,3 +2,82 @@
...
+    width: 400px;
+    background-color: rgba(255,255,255,0.13);
+    position: absolute;
+    transform: translate(-50%,-50%);
+    top: 50%;
+    left: 50%;
+    border: 2px solid blueviolet;
+    border-radius: 10px;
+    backdrop-filter: blur(10px);
+    box-shadow: 0 0 40px rgba(95, 84, 177, 0.6);
```
Comment (AshaPatel216) [inline src/styles.scss:34]: Use color code in next PR

File: src/styles.scss [hunk 43]
```diff
@@ -2,3 +2,82 @@
...
+    padding: 30px 15px;
+    font-family: ‘Poppins’,sans-serif;
+    color: blueviolet;
```
Comment (AshaPatel216) [inline src/styles.scss:37]: same as above

File: src/app/login-signup/login-signup.module.ts (hunk 6 above)
Comment (AshaPatel216) [reply src/app/login-signup/login-signup.module.ts]: Why this is not removed yet?

File: src/app/login-signup/login/login.component.ts [hunk 44]
```diff
@@ -1,10 +1,61 @@
...
+  loginForm!: FormGroup;
+  signupUsers: User[];
+
+  constructor(private router: Router) {
+    this.signupUsers = [];
+   }
+
+  ngOnInit():void {
+    const localData = localStorage.getItem('signupUsers');
+    if(localData != null) {
```
Comment (AshaPatel216) [inline src/app/login-signup/login/login.component.ts]: Use strong type for comparision. != -> !==

Comment (AshaPatel216) [reply src/app/login-signup/login/login.component.ts]: Please follow this now onwards

File: src/styles.scss [hunk 45]
```diff
@@ -2,3 +2,100 @@
...
+
+//email div
+.email {
+    margin-top: 15px ;
+    margin-bottom: 15px;
+}
+//password div
+.password {
+    margin-top: 15px ;
+    margin-bottom: 15px;
```
Comment (AshaPatel216) [inline src/styles.scss]: email and password class has a common style. Please use common class of bootstrap to give margin and padding. remove unnecessary class. https://getbootstrap.com/docs/4.0/utilities/spacing/

File: src/styles.scss [hunk 46]
```diff
@@ -2,3 +2,100 @@
...
+}
+//confirm-password div
+.confirm-password { 
+    margin-bottom: 15px;
```
Comment (AshaPatel216) [inline src/styles.scss]: Same as above

File: src/styles.scss [hunk 47]
```diff
@@ -2,3 +2,100 @@
...
+
+//h1 styling
+h1 {
+    text-align: center;
+    margin-top: 10px ;
+}
+
+//welcome page styling
+.welcome-page {
+    text-align: center;
```
Comment (AshaPatel216) [inline src/styles.scss]: Use text align class from bootstrap:  https://getbootstrap.com/docs/4.0/utilities/text/

File: src/app/login-signup/login/login.component.ts (hunk 44 above)
Comment (1810Nikita) [reply src/app/login-signup/login/login.component.ts]: Changes are done

File: src/styles.scss (hunk 43 above)
Comment (1810Nikita) [reply src/styles.scss:37]: I will do changes in next PR

File: src/styles.scss (hunk 42 above)
Comment (1810Nikita) [reply src/styles.scss:34]: I will do changes in next PR

File: src/styles.scss (hunk 41 above)
Comment (1810Nikita) [reply src/styles.scss:36]: I will do changes in next PR

File: src/styles.scss (hunk 40 above)
Comment (1810Nikita) [reply src/styles.scss]: Changes are done

File: src/styles.scss (hunk 39 above)
Comment (1810Nikita) [reply src/styles.scss]: Changes are done.

File: src/app/styles/components/button.scss (hunk 38 above)
Comment (1810Nikita) [reply src/app/styles/components/button.scss:17]: I will do changes in next PR

File: src/app/styles/components/button.scss (hunk 37 above)
Comment (1810Nikita) [reply src/app/styles/components/button.scss:18]: I will do changes in next PR

File: src/app/styles/components/button.scss (hunk 36 above)
Comment (1810Nikita) [reply src/app/styles/components/button.scss]: Changes are done

File: src/app/login-signup/register/register.component.ts (hunk 35 above)
Comment (1810Nikita) [reply src/app/login-signup/register/register.component.ts]: Changes are done

File: src/app/login-signup/register/register.component.ts (hunk 34 above)
Comment (1810Nikita) [reply src/app/login-signup/register/register.component.ts]: Changes are done

File: src/app/login-signup/register/register.component.ts (hunk 33 above)
Comment (1810Nikita) [reply src/app/login-signup/register/register.component.ts:6]: I will do changes in next PR

File: src/app/login-signup/register/register.component.html (hunk 32 above)
Comment (1810Nikita) [reply src/app/login-signup/register/register.component.html]: Changes are done

File: src/app/login-signup/login/login.component.ts (hunk 31 above)
Comment (1810Nikita) [reply src/app/login-signup/login/login.component.ts]: Changes are done

File: src/app/login-signup/login/login.component.ts (hunk 30 above)
Comment (1810Nikita) [reply src/app/login-signup/login/login.component.ts]: Changes are done.

File: src/app/login-signup/login/login.component.ts (hunk 29 above)
Comment (1810Nikita) [reply src/app/login-signup/login/login.component.ts:41]: Changes are done

File: src/app/login-signup/login/login.component.ts (hunk 28 above)
Comment (1810Nikita) [reply src/app/login-signup/login/login.component.ts]: Changes are done

File: src/app/login-signup/login/login.component.ts (hunk 27 above)
Comment (1810Nikita) [reply src/app/login-signup/login/login.component.ts:8]: I will do changes in next PR

File: src/app/login-signup/login/login.component.html (hunk 26 above)
Comment (1810Nikita) [reply src/app/login-signup/login/login.component.html:20]: Changes are done.

File: src/app/login-signup/login/login.component.html (hunk 25 above)
Comment (1810Nikita) [reply src/app/login-signup/login/login.component.html]: Changes are done

File: src/app/login-signup/login/login.component.html (hunk 24 above)
Comment (1810Nikita) [reply src/app/login-signup/login/login.component.html]: Changes are done

File: src/app/login-signup/login/login.component.html (hunk 23 above)
Comment (1810Nikita) [reply src/app/login-signup/login/login.component.html]: Changes are done.

File: src/app/app.component.ts (hunk 22 above)
Comment (1810Nikita) [reply src/app/app.component.ts]: Changes are done

File: src/app/app.component.scss (hunk 1 above)
Comment (1810Nikita) [reply src/app/app.component.scss]: Changes are done

File: src/app/app.component.scss (hunk 2 above)
Comment (1810Nikita) [reply src/app/app.component.scss]: I will do changes in next PR

File: src/app/app.component.ts (hunk 3 above)
Comment (1810Nikita) [reply src/app/app.component.ts]: Changes are done

File: src/app/app.component.ts (hunk 4 above)
Comment (1810Nikita) [reply src/app/app.component.ts:16]: Changes are done

File: src/app/login-signup/login-signup.component.html (hunk 5 above)
Comment (1810Nikita) [reply src/app/login-signup/login-signup.component.html]: Changes are done

File: src/app/login-signup/login-signup.module.ts (hunk 6 above)
Comment (1810Nikita) [reply src/app/login-signup/login-signup.module.ts]: Changes are done

File: src/app/login-signup/login-signup.module.ts (hunk 7 above)
Comment (1810Nikita) [reply src/app/login-signup/login-signup.module.ts]: Changes are done

File: src/app/login-signup/login/login.component.html (hunk 8 above)
Comment (1810Nikita) [reply src/app/login-signup/login/login.component.html]: Changes are done

File: src/app/login-signup/login/login.component.html (hunk 9 above)
Comment (1810Nikita) [reply src/app/login-signup/login/login.component.html]: Changes are done

File: src/app/login-signup/login/login.component.html (hunk 10 above)
Comment (1810Nikita) [reply src/app/login-signup/login/login.component.html]: Changes are done

File: src/app/login-signup/login/login.component.ts (hunk 11 above)
Comment (1810Nikita) [reply src/app/login-signup/login/login.component.ts]: Changes are done

File: src/app/login-signup/login/login.component.ts (hunk 12 above)
Comment (1810Nikita) [reply src/app/login-signup/login/login.component.ts]: Changes are done

File: src/app/login-signup/login/login.component.ts (hunk 13 above)
Comment (1810Nikita) [reply src/app/login-signup/login/login.component.ts]: Changes are done

File: src/app/login-signup/login/login.component.ts (hunk 14 above)
Comment (1810Nikita) [reply src/app/login-signup/login/login.component.ts]: Changes are done

File: src/app/login-signup/login/login.component.ts (hunk 15 above)
Comment (1810Nikita) [reply src/app/login-signup/login/login.component.ts]: Changes are done

File: src/app/login-signup/login/login.component.ts (hunk 16 above)
Comment (1810Nikita) [reply src/app/login-signup/login/login.component.ts]: Changes are done

File: src/app/login-signup/register/register.component.html (hunk 17 above)
Comment (1810Nikita) [reply src/app/login-signup/register/register.component.html]: Changes are done

File: src/app/login-signup/register/register.component.html (hunk 18 above)
Comment (1810Nikita) [reply src/app/login-signup/register/register.component.html]: Changes are done

File: src/app/login-signup/register/register.component.ts (hunk 19 above)
Comment (1810Nikita) [reply src/app/login-signup/register/register.component.ts]: Changes are done

File: src/app/app.component.html (hunk 20 above)
Comment (1810Nikita) [reply src/app/app.component.html]: Changes are done

File: src/app/app.component.html (hunk 21 above)
Comment (1810Nikita) [reply src/app/app.component.html]: Changes are done

File: src/styles.scss (hunk 45 above)
Comment (1810Nikita) [reply src/styles.scss]: Changes are done

File: src/styles.scss (hunk 46 above)
Comment (1810Nikita) [reply src/styles.scss]: Changes are done

File: src/styles.scss (hunk 47 above)
Comment (1810Nikita) [reply src/styles.scss]: Changes are done

**Review by AshaPatel216 — APPROVED** [review]

**Pull request merged**